apollo-federation/src/query_graph/condition_resolver.rs

```diff
@@ -58,6 +58,30 @@ pub(crate) enum ConditionResolution {
     },
 }
 
+impl std::fmt::Display for ConditionResolution {
```

Comment:
This is small, but you can derive a `Display` impl using [`derive_more`](https://docs.rs/derive_more/latest/derive_more/derive.Display.html#shared-enum-format).

Reply:
Cool! I could've used that. But, I already wrote this. So, I'll remember it next time!